### ISSUE TITLE
RPM version of 2.4.46 don't update olders

### ISSUE THREAD

**Issue by bvaladon** [ltb-project/openldap-rpm#22]
Bonjour Clement, 

Lors de la MAJ de la version 2.4.46, openldap ne repart pas.

Lors de la MAJ : 
```
warning: /etc/default/slapd saved as /etc/default/slapd.rpmsave
Job for slapd.service failed because the control process exited with error code. See "systemctl status slapd.service" and "journalctl -xe" for details.

-- Unit slapd.service has begun shutting down.
Apr 06 15:37:30 bddsys1 slapd[13805]: [INFO] Using /etc/default/slapd for configuration
Apr 06 15:37:30 bddsys1 slapd[13795]: slapd: [INFO] Using /etc/default/slapd for configuration
Apr 06 15:37:30 bddsys1 slapd[13807]: [INFO] Halting OpenLDAP...
Apr 06 15:37:30 bddsys1 slapd[13795]: slapd: [INFO] Halting OpenLDAP...
Apr 06 15:37:30 bddsys1 slapd[1329]: daemon: shutdown requested and initiated.
Apr 06 15:37:30 bddsys1 slapd[1329]: conn=1000 fd=13 closed (slapd shutdown)
Apr 06 15:37:30 bddsys1 slapd[1329]: slapd shutdown: waiting for 0 operations/tasks to finish
Apr 06 15:37:30 bddsys1 slapd[1329]: slapd stopped.
Apr 06 15:37:31 bddsys1 slapd[13811]: [OK] OpenLDAP stopped after 1 seconds
Apr 06 15:37:31 bddsys1 slapd[13795]: slapd: [OK] OpenLDAP stopped after 1 seconds
Apr 06 15:37:31 bddsys1 slapd[13812]: [INFO] No data backup done
Apr 06 15:37:31 bddsys1 slapd[13795]: slapd: [INFO] No data backup done
Apr 06 15:37:31 bddsys1 systemd[1]: Stopped LSB: OpenLDAP.
-- Subject: Unit slapd.service has finished shutting down
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
--
-- Unit slapd.service has finished shutting down.
Apr 06 15:37:33 bddsys1 systemd[1]: Reloading.
Apr 06 15:37:33 bddsys1 systemd[1]: [/usr/lib/systemd/system/slapd.service:10] Unknown lvalue 'ExecRestart' in section 'Service'
Apr 06 15:37:33 bddsys1 yum[13687]: Updated: openldap-ltb-2.4.46-1.el7.x86_64
Apr 06 15:37:34 bddsys1 systemd[1]: [/usr/lib/systemd/system/slapd.service:10] Unknown lvalue 'ExecRestart' in section 'Service'
Apr 06 15:37:34 bddsys1 systemd[1]: Starting OpenLDAP LTB startup script...
-- Subject: Unit slapd.service has begun start-up
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
--
-- Unit slapd.service has begun starting up.
Apr 06 15:37:34 bddsys1 slapd-cli[13854]: [INFO] Using /usr/local/openldap/etc/openldap/slapd-cli.conf for configuration
Apr 06 15:37:34 bddsys1 slapd-cli[13844]: slapd-cli: [INFO] Using /usr/local/openldap/etc/openldap/slapd-cli.conf for configuration
Apr 06 15:37:34 bddsys1 slapd-cli[13859]: [INFO] Create LDAPI socket dir /var/run/slapd
Apr 06 15:37:34 bddsys1 slapd-cli[13844]: slapd-cli: [INFO] Create LDAPI socket dir /var/run/slapd
Apr 06 15:37:34 bddsys1 slapd-cli[13863]: [INFO] Launching OpenLDAP configuration test...
Apr 06 15:37:34 bddsys1 slapd-cli[13844]: slapd-cli: [INFO] Launching OpenLDAP configuration test...
Apr 06 15:37:34 bddsys1 slapd-cli[13865]: [ALERT] OpenLDAP configuration test failed
Apr 06 15:37:34 bddsys1 slapd-cli[13844]: slapd-cli: [ALERT] OpenLDAP configuration test failed
Apr 06 15:37:34 bddsys1 systemd[1]: slapd.service: control process exited, code=exited status=1
Apr 06 15:37:34 bddsys1 systemd[1]: Failed to start OpenLDAP LTB startup script.
-- Subject: Unit slapd.service has failed
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
--
-- Unit slapd.service has failed.
--
-- The result is failed.
Apr 06 15:37:34 bddsys1 systemd[1]: Unit slapd.service entered failed state.
```

**Comment by coudot** [comment]
Hello Bruno,

the config test fails, that's why the daemon does not restart.

Try: chown -R ldap:ldap  /usr/local/etc/openldap/slapd.d

**Comment by coudot** [comment]
Fixed in #23
